### ISSUE TITLE
Last failing unit test on Oracle and php 5.6

### ISSUE THREAD

**Issue by DeepDiver1975** [doctrine/dbal#2281]
https://ci.owncloud.org/job/doctrine-dbal-oracle/15/PHP=system,label=SLAVE/console

```
There was 1 failure:

1) Doctrine\Tests\DBAL\Functional\DataAccessTest::testDateArithmetics
Date difference should be approx. -2198 days.
Failed asserting that -2196 matches expected -2198.0.

/ssd/jenkins/workspace/doctrine-dbal-oracle/PHP/system/label/SLAVE/tests/Doctrine/Tests/DBAL/Functional/DataAccessTest.php:524
```

**Comment by deeky666** [comment]
I think I remember that one. IIRC there are some issues with those tests on some platform. Not sure whether we can solve this or need to skip.

**Comment by Deltachaos** [comment]
@DeepDiver1975 i guess that this would not be solved by #2365 right? The actual problem seems to be that the ``getDateDiffExpression`` method is returning a function that is not working correctly right?

**Comment by lcobucci** [comment]
@Ocramius @deeky666 is this still relevant since we dropped PHP 5 on `master`?

**Comment by lcobucci** [comment]
I mean it seems to not be related to PHP at all but rather formats

**Comment by Ocramius** [comment]
Closing this: PHP 5 will no longer be supported by DBAL `2.6`, and patches that aren't security-only should not be merged into `2.5` anymore.
